### ISSUE TITLE
Bad pipeline sequence with proj_create_crs_to_crs() and PROJ strings

### ISSUE THREAD

**Issue by moulinfr** [OSGeo/PROJ#1464]
Call to:
`proj_create_crs_to_crs(pj_ctx,
                                   "+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs +over", 
                                   "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0 +k=1.0 +units=m +nadgrids=@null +wktext +no_defs +over", NULL);`
fails and error string returns 'Bad sequence...' for PROJ.4 5.2.0 (definitions ported from PROJ.4 4.9.0)

The same call using respectively "epsg:4326" and "epsg:3857" instead of PROJ strings works but I cannot use "+over" which is required I my use case. 

Maybe this is related to #1220? 

Please, could you confirm that it is the expected behavior and what would be the right approach?

Thank you very much.

F.M.

**Comment by rouault** [comment]
This has been fixed in PROJ 6.0
```
$ echo "180 45" | ~/proj/install-proj-6.0.0/bin/cs2cs "+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs +over"  "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0 +k=1.0 +units=m +nadgrids=@null +wktext +no_defs +over"
20037508.34	5621521.49 0.00

$ echo "-180 45" | ~/proj/install-proj-6.0.0/bin/cs2cs "+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs +over"  "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0 +k=1.0 +units=m +nadgrids=@null +wktext +no_defs +over"
-20037508.34	5621521.49 0.00
```

cs2cs in PROJ 6 uses proj_create_crs_to_crs()

**Comment by moulinfr** [comment]
Merci beaucoup ! Many thanks for your quick answer. I'll try shift to this version as soon as I can and will provide you with a feedback.
